### PR TITLE
Added harvest documentation for tiered offices and departments

### PR DESCRIPTION
**Pull request by jdnguyen**
https://greenhouseio.atlassian.net/browse/GREEN-4955

### CODE REVIEW

File: source/includes/harvest/_departments.md [hunk 1]
```diff
@@ -33,11 +35,15 @@ curl 'https://harvest.greenhouse.io/v1/departments'
...
-    "name": "Technology"
+    "name": "Technology",
+    "parent_id": 5,
+    "child_ids": [11, 12]
   },
   {
     "id": 2,
-    "name": "Administration"
+    "name": "Administration",
+    "parent_id": nil,
```
Comment (jcmuller) [inline source/includes/harvest/_departments.md]: `null`

File: source/includes/harvest/_offices.md [hunk 2]
```diff
@@ -35,20 +37,26 @@ curl 'https://harvest.greenhouse.io/v1/offices'
...
+    "parent_id": 180,
+    "child_ids": [190, 195],
     "location": {
       "name": "San Francisco, CA"
     }
   },
   {
     "id": 344,
     "name": "Bangkok",
+    "parent_id": nil,
```
Comment (jcmuller) [inline source/includes/harvest/_offices.md]: Same in this file.

**Comment by jcmuller** [comment]
Should we also add a tree sample?

**Comment by jdnguyen** [comment]
@jcmuller think our docs are ready to go out?

**Pull request merged**